etl/embedding_model_pipelines/embedding_model_pipeline.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+from langchain.docstore.document import Document
+from langchain_core.vectorstores import VectorStore
+
+
+class EmbeddingModelPipeline(ABC):
```

Comment:
Nit: it's really an `EmbeddingPipeline` and not an `EmbeddingModelPipeline`. It uses an embedding model.

Reply:
That is true. This class is now `EmbeddingPipeline`.